source/brailleTables.py

```diff
@@ -76,7 +76,7 @@ def listTables():
 	"mn-MN.utb":"mn-MN-g1.utb",
 	"nl-BE-g1.ctb":"nl-BE-g0.utb",
 	"nl-NL-g1.ctb":"nl-NL-g0.utb",
-	"no-no.ctb":"no-no-comp8.ctb",

```

Comment:
If you remove this line, people with an old version of NVDA upgrading to a newer version might  get stuck, since no-no.ctb might not exist any more.
Please add `"no-no.ctb":"no-no-comp8.itb",`